pkg/controller/hibernation/aws_actuator.go

```diff
@@ -96,34 +96,34 @@ func (a *awsActuator) StartMachines(cd *hivev1.ClusterDeployment, c client.Clien
 
 // MachinesRunning will return true if the machines associated with the given
 // ClusterDeployment are in a running state.
-func (a *awsActuator) MachinesRunning(cd *hivev1.ClusterDeployment, c client.Client, logger log.FieldLogger) (bool, error) {
+func (a *awsActuator) MachinesRunning(cd *hivev1.ClusterDeployment, c client.Client, logger log.FieldLogger) (bool, []string, error) {
 	logger = logger.WithField("cloud", "aws")
 	logger.Infof("checking whether machines are running")
 	awsClient, err := a.awsClientFn(cd, c, logger)
 	if err != nil {
-		return false, err
+		return false, nil, err
 	}
 	instanceIDs, err := getClusterInstanceIDs(cd, awsClient, notRunningStates, logger)
 	if err != nil {
-		return false, err
+		return false, nil, err
 	}
-	return len(instanceIDs) == 0, nil
+	return len(instanceIDs) == 0, aws.StringValueSlice(instanceIDs), nil
```

Comment:
Note: `StringValueSlice` will never return `nil`. If `instanceIDs` is `nil` or zero length, it will return `[]string{}`. Same applies to all impls for both funcs.